django/db/backends/dummy/base.py

```diff
@@ -83,4 +83,4 @@ def __init__(self, *args, **kwargs):
         self.validation = BaseDatabaseValidation(self)
 
     def is_usable(self):
-        return True
+        return False
```

Comment:
I'm not sure about other implications this change has. Tests seem to be passing but I don't know we actually ever check for this to be `True` on the dummy back-end?